common/ipc-client.c

```diff
@@ -69,6 +69,18 @@ int ipc_open_socket(const char *socket_path) {
 	return socketfd;
 }
 
+bool ipc_set_recv_timeout(int socketfd, struct timeval tv) {
+	int result = setsockopt(socketfd, SOL_SOCKET, SO_RCVTIMEO,
+			(const char*)&tv, sizeof(tv));
```

Comment:
`man 3p setsockopt` says the value is a `const void *`, so I don't think a cast is necessary.

Reply:
Removed cast. Sorry about that, I searched for how to set a timeout. Apparently Windows has the argument as `const char *` instead of `const void *` like every other platform so I was seeing the snippet with cast for compatibility. I forgot to check if the cast was actually needed.